test/functional/Valhalla/playlist.xml

```diff
@@ -292,6 +292,15 @@
 	</test>
 	<test>
 		<testCaseName>ValhallaAttributeTests</testCaseName>
+		<variations>
+			<variation>-Xint</variation>
+			<variation>-Xint -Xgcpolicy:optthruput -XX:ValueTypeFlatteningThreshold=99999 -XX:-EnableArrayFlattening</variation>
```

Comment:
What is the purpose of having optthruput here?

Reply:
>What is the purpose of having optthruput here?

I don't think the GC policy matters for these attribute tests. I just copied the variations from other tests. I'll remove optthruput in the next commit and use the default GC policy.

Reply:
Addressed in 32f6437